src/App.tsx

```diff
@@ -1,156 +1,86 @@
+/* eslint-disable react-hooks/rules-of-hooks */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+
+import { Select } from './types/Select';
+import { Todo } from './types/Todo';
+import { client } from './utils/fetchClient';
 
-const USER_ID = 0;
+const USER_ID = 10364;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [select, setSelect] = useState(Select.All);
+  const [errorMessage, setErrorMessage] = useState('');
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const todoUrlEnd = '/todos?userId=10364';
+
+  const createTodo = (data: Todo) => {
+    client.post(todoUrlEnd, data);
+  };
+
+  const getTodo = async () => {
```

Comment:
use useCallback here
```suggestion
  const getTodos = async () => {
```